packages/framer-motion/src/render/VisualElement.ts

```diff
@@ -45,7 +45,6 @@ const propEventHandlers = [
     "AnimationStart",
     "AnimationComplete",
     "Update",
-    "Unmount",
```

Comment:
Do we also need to cleanup the interface here https://github.com/framer/motion/blob/2eb37fc5fac7bd94ff57492f6e4164e93d787b50/packages/framer-motion/src/render/types.ts#L137?